src/Datadog.Trace.ClrProfiler.Managed/Integrations/GraphQLIntegration.cs

```diff
@@ -254,7 +254,8 @@ private static Scope CreateScopeFromValidate(object document)
                 scope = tracer.StartActive(ValidateOperationName, serviceName: serviceName);
                 var span = scope.Span;
                 span.Type = SpanTypes.GraphQL;
-
+                span.SetTag(Tags.SpanKind, SpanKinds.Server);
```

Comment:
This is also missing in CreateScopeFromExecuteAsync. Can you add that there too?

Reply:
Done